kubernetes_asyncio/watch/watch.py

```diff
@@ -178,3 +182,11 @@ def stream(self, func, *args, **kwargs):
         self.resp = None
```

Comment:
This line can be removed now.

Reply:
User may reuse the `Watch` object to start another stream request, so this PR checks the `resp` property in `stream()`